http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/PersistentDiskComponent.scala

```diff
@@ -0,0 +1,150 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package db
+
+import java.time.Instant
+
+import cats.implicits._
+import org.broadinstitute.dsde.workbench.google2.{DiskName, ZoneName}
+import org.broadinstitute.dsde.workbench.leonardo.db.LeoProfile.api._
+import org.broadinstitute.dsde.workbench.leonardo.db.LeoProfile.mappedColumnImplicits._
+import org.broadinstitute.dsde.workbench.leonardo.db.LeoProfile.{dummyDate, unmarshalDestroyedDate}
+import org.broadinstitute.dsde.workbench.model.WorkbenchEmail
+import org.broadinstitute.dsde.workbench.model.google.GoogleProject
+
+import scala.concurrent.ExecutionContext
+
+final case class PersistentDiskRecord(id: Long,
+                                      googleProject: GoogleProject,
+                                      zone: ZoneName,
+                                      name: DiskName,
+                                      googleId: Option[GoogleId],
+                                      samResourceId: DiskSamResourceId,
+                                      status: DiskStatus,
+                                      creator: WorkbenchEmail,
+                                      createdDate: Instant,
+                                      destroyedDate: Instant,
+                                      dateAccessed: Instant,
```

Comment:
can we move these fields into `AuditInfo`?

Reply:
We could.. but this lets us write `<> (PersistentDiskRecord.tupled, PersistentDiskRecord.unapply)` below. I kind of see this class as a direct mapping to columns in the database, whereas the `PersistentDisk` class has more structure.

Reply:
I don't feel strongly either way..But if we see this as direct mapping, it's easy to run into 22 fields limit tho.